nerfstudio/data/datamanagers/base_datamanager.py

```diff
@@ -283,6 +287,11 @@ class VanillaDataManagerConfig(InstantiateConfig):
     """The scale factor for scaling spatial data such as images, mask, semantics
     along with relevant information about camera intrinsics
     """
+    pixel_sampling: Literal["uniform", "patch"] = "uniform"
+    """How pixels are sampling in space. If uniform, pixels are evenly sampled between pixels
+    """
+    pixel_patch_size: int = -1
+    """Size of patch to sample from. Only used if using patch-based pixel sampling."""
```

Comment:
Can this be reduce to one argument, `patch_size`? If the value is 1, it using the pixel sampler, if >= 1 it uses the patch sampler.